controllers/activemqartemis_reconciler.go

```diff
@@ -2302,7 +2182,7 @@ func createExtraConfigmapsAndSecretsVolumeMounts(brokerContainer *corev1.Contain
 	if len(secrets) > 0 {
 		for _, secret := range secrets {
 			if secret == "" {
-				clog.Info("No Secret name specified, ignore", "Secret", secret)
+				r.log.V(2).Info("No Secret name specified, ignore", "Secret", secret)
```

Comment:
V(2).Info means that users need to set `--zap-log-level` >= 2 because `--zap-log-level=debug` only prints Info levels <= 1. Is that on purpose?

Reply:
yes I was thinking about the rules at the moment. Making it level 2 because it prints the whole secret object.
rather that a simple type var. (same rule apply to printing a whole statefulset, deployment etc).
But we need a more comprehensive set of logging rules to follow. I'll be thinking of it.